src/components/Settings/SaveSearchItem.vue

```diff
@@ -47,7 +47,7 @@ export default {
 	data() {
 		return {
 			showAlerts: this.savedSearch?.isAlert,
-			deleteSearch: () => []
+			totalCount: this.savedSearch?.loanCriteriaSearch?.filters?.loanIds
```

Comment:
This should be initialized as `0`. But you can leave it for now, I'll clean it up.

FYI @dyersituations anything to do with showing the the loan count I'll go through and clean up as I find a better way to query for how many loans are available for each saved search.